cheat.js

```diff
@@ -0,0 +1,2 @@
+// This severity is too high for any use other than development.
+var ses = {maxAcceptableSeverityName: 'NEW_SYMPTOM'};
```

Comment:
Elsewhere I revised this to

var ses = ses || {};
// This severity is too high for any use other than development.
ses.maxAcceptableSeverityName = 'NEW_SYMPTOM';

which I like better because, if there already is a ses object, it does not wipe it out.